readme.md

```diff
@@ -12,33 +12,36 @@
 ## Authors
 
 - [@TrivCodez](https://github.com/TrivCodez/)
+
 ## Contributing
 
 Contributions are always welcome!
 
 See `contributing.md` for ways to get started.
+
 ## FAQ
 
 #### What is this repo?
 
-This repo is a repo containing ways you can use chatgpt for anything! We will provide high-quality prompts for each category such as development, art ideas, etc. 
+This repo is a repo containing ways you can use chatgpt for anything! We will provide high-quality prompts for each category such as development, art ideas, etc.
 
 #### Who owns this repo?
 
 - [@TrivCodez](https://github.com/TrivCodez/)
 
 However we also accept Contributions, once you have contributed you will be shown in the Contributions list!
+
 ## Feedback
 
-If you have any feedback, please reach out by creating a issue by [clicking here](https://github.com/TrivCodez/chatgpt-prompts/issues/new). Please `⭐️` this repo as well because it helps this repo be recognized by other users! 
+If you have any feedback, please reach out by creating a issue by [clicking here](https://github.com/TrivCodez/chatgpt-prompts/issues/new). Please `⭐️` this repo as well because it helps this repo be recognized by other users!
```

Comment:
```
If you have any feedback, please reach out by [creating an issue here](https://github.com/TrivCodez/chatgpt-prompts/issues/new). Please `⭐️` this repo as well because it helps this repo be recognized by other users!
```
repeating this formula here..